src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -3179,6 +3255,7 @@ public void testTLSClientRevokedServerCert() throws Exception {
 
   @Test
   //Client specifies cert that the server does not trust via a revoked certificate of the CA
+  @Ignore // this test must be reviewed - it was "passing" when "fail" was expected
```

Comment:
Sure!

I've added an [assert](https://github.com/eclipse/vert.x/pull/1276/files#diff-19c82d5fdde982f60afb99476e34f478R3354) to the `testTLS()` method to ensure that the `shouldPass` value is indeed true when it reaches that work-flow.
Looking at the definition of that particular test case, it was supposed to "fail" since:
1. With a revoked client certificate, the channel will not be established
2. `shouldPass = false`
But, the test was following a "success" work-flow, which lead to that new assert to fail.